src/plugins/intel_cpu/src/nodes/paged_attn.cpp

```diff
@@ -152,18 +152,32 @@ void PagedAttention::execute(dnnl::stream strm) {
         inputs[i] = getSrcMemoryAtPort(i);
     }
 
-    const auto& queryDims = inputs[0]->getStaticDims();
+    auto outDims = inputs[0]->getStaticDims();
+    const auto& keyDims = inputs[1]->getStaticDims();
+    const auto& valueDims = inputs[2]->getStaticDims();
+    // value head_size may be not same with key
+    if (keyDims[1] != valueDims[1]) {
+        // The outDims[1] should be `num_heads * v_head_size`, it can be got from:
+        // because:
+        //   q: query_ps[1] = num_heads * head_size
+        //   v: key_ps[1] = num_kv_heads * head_size
```

Comment:
typo in the comment, is k should has head_size not v_head_size

Reply:
Fixed.